### DISCUSSION TITLE
Are wind chill and heat index mutually exclusive?

### DISCUSSION THREAD

**Discussion by sdennstaedt** [weather-gov/api#393] [Q&A]
For the latest observations, can I assume that both of these values are mutually exclusive?  That is, you can't have a value for both heatIndex and windChill.  One will be null.  Can they both be null?  

"windChill": {
            "value": null,
            "unitCode": "unit:degC",
            "qualityControl": "qc:V"
        },
        "heatIndex": {
            "value": 37.114885778912878,
            "unitCode": "unit:degC",
            "qualityControl": "qc:V"

**Comment by StephenClouse** [comment]
Yes and yes.

[Heat index](https://www.wpc.ncep.noaa.gov/html/heatindex_equation.shtml) is only valid when the real temperature is above 70F.
[Wind chill](https://www.weather.gov/safety/cold-wind-chill-chart) is only valid between -50 and 50F.

The values will always be null outside of these temperatures.
